src/net/sourceforge/kolmafia/request/coinmaster/shop/DedigitizerRequest.java

```diff
@@ -48,11 +48,10 @@ public static void parseResponse(final String location, final String responseTex
   }
 
   public static String accessible() {
-    int serverRoomKey = SERVER_ROOM_KEY.getCount(KoLConstants.inventory);
-    if (serverRoomKey == 0) {
-      return "You don't have a server room key in inventory.";
+    if (Preferences.getBoolean("crAlways") || Preferences.getBoolean("_crToday")) {
+      return null;
     }
-    return null;
+    return "You don't have a server room key in inventory.";
```

Comment:
perhaps more like "You can't access the server room."?

Reply:
Sure.